tfhe/src/core_crypto/algorithms/test/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -93,11 +93,7 @@ where
 
     let mut rsc = TestResources::new();
 
-    let f = |x: Scalar| {
-        x.wrapping_mul(Scalar::TWO)
-            .wrapping_sub(Scalar::ONE)
-            .wrapping_rem(msg_modulus)
-    };
+    let f = |x: Scalar| x;
 
     let delta: Scalar = encoding_with_padding / msg_modulus;
     let mut msg = msg_modulus;
```

Comment:
Could be initialized to `carry_modulus * msg_modulus`
Comment applies to other files

Reply:
Shouldn't we not see carry and message concepts in core_crypto ?

Reply:
strictly the carry/msg vision is a shortint vision only required when combining cleartext multiplications or ciphertexts additions with the KS + PBS, here I'm using the full plaintext space without looping operations, I don't think it's an issue as is, though I could add the carry information to parameters but keep working on the whole message space, but I don't know if we really need that at the moment, so I would probably say it's a backlog issue, wdyt ?

Reply:
A backlog issue is fine for me

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/351